lib/rialto/etl/logging.rb

```diff
@@ -10,10 +10,25 @@ module Logging
 
       def logger
         @logger ||= Yell.new do |logger|
-          logger.adapter :datefile, 'rialto_etl.log', level: 'lte.warn' # anything lower or equal to :warn
-          logger.adapter :datefile, 'rialto_etl_error.log', level: 'gte.error' # anything greater or equal to :error
+          logger.adapter :datefile, info_log, level: 'lte.warn' # anything lower or equal to :warn
+          logger.adapter :datefile, error_log, level: 'gte.error' # anything greater or equal to :error
         end
       end
+
+      private
+
+      def log_path
+        return './log/' unless Settings.key?(:log_path)
+        Settings.log_path
+      end
+
+      def info_log
+        log_path + 'rialto_etl.log'
```

Comment:
I have a minor (key word) preference for `File.join(log_path, 'WHATEVER')` here and below rather than using string concat.

Reply:
I knew there was something better... Thanks!